### PR TITLE
Fixed the footer to be the same height on all pages

### PR DESCRIPTION
**Pull request by Justyne**
Signed-off-by: Justyne justyne@gmail.com

### CODE REVIEW

File: other/upgrade.php [hunk 1]
```diff
@@ -3837,6 +3838,7 @@ function template_upgrade_below()
 			</div>
 			</div>
 		</div>
+		</div>
```
Comment (XinYenFon) [inline other/upgrade.php:3841]: extra tab here.

File: other/install.php [hunk 2]
```diff
@@ -2112,7 +2112,7 @@ function template_install_above()
 		', $txt['lang_rtl'] == true ? '<link rel="stylesheet" href="Themes/default/css/rtl.css?alp21">' : '' , '
 		<script src="Themes/default/scripts/script.js"></script>
 	</head>
-	<body>
+	<body><div id="footerfix">
```
Comment (XinYenFon) [inline other/install.php:2115]: pick one style :P either do body-div or body(new-line)div

**Comment by XinYenFon** [comment]
You forgot to sign-off last commit. If you fix them I think we are good to go here (maybe pick new ID instead footerfix).

**Comment by XinYenFon** [comment]
Also I have to admit, I'm not really fan of adding extra container to template system.

**Comment by Arantor** [comment]
If it were possible to fix any other way than adding new divs, it might have been doable, but having a footer that's half the page height looks terrible.

As for the adding of tags, that was mostly because no-one really wants to go through and fix all the indentation everywhere (when it's already not consistent as it is)

**Comment by XinYenFon** [comment]
seems so, I think this is good to go :)

**Pull request merged**